src/components/atoms/button/button.html

```diff
@@ -0,0 +1 @@
+<a href="a" class="button">btn-text</a>
```

Comment:
```suggestion
<a href="#" class="button">btn-text</a>
```
ダミーのリンクは `#` としておくのがおすすめです！
現状の書き方だとボタンを押すと変なページに遷移してしまいますが、 `#` だとidへのページ内リンクとみなされてページ内にとどまるという挙動になります。